mozilla/kuma
# Fix bug 1119397: Manage remaining straightforward dependencies with Bower

Testing this change:

1. `vagrant provision`
2. `vagrant ssh`
3. Verify that `./manage.py collectstatic --noinput` works
4. Browse around the site and verify that things still behave as expected (I don't expect anything too weird to happen here since the files were only renamed)
